stacks/apps/apple-api-bridge.yml

```diff
@@ -0,0 +1,349 @@
+# stacks/apps/apple-api-bridge.yml
```

Comment:
For consistency, this file should probably be called `dovetail-apple-api-bridge.yml`. Some of the oldest templates in the Dovetail family don't follow that pattern, but only because they predate it.